test/rekt/resources/parallel/parallel.go

```diff
@@ -102,26 +108,31 @@ func WithSubscriberAt(index int, ref *duckv1.KReference, uri string) manifest.Cf
 		}
 		subscriber := branch["subscriber"].(map[string]interface{})
 
```

Comment:
Maybe check if d is nil here as well just like other functions like `WithFilterAt` do?

Reply:
`d` nil, while possible, it doesn't make sense and this is test code, so it's ok to panic in that case